Resources/Prototypes/Traits/skills.yml

```diff
@@ -43,19 +47,16 @@
       jobs:
         - Borg
         - MedicalBorg
-    - !type:CharacterTraitRequirement
-      inverted: true
-      traits:
-        - LightweightDrunk
-        - LiquorLifeline
```

Comment:
Once again I don't understand why would you remove requirements here. Applies to the whole file.

Reply:
Those requirements are very hard to maintain and it's easy to make the mistake of not updating existing requirements when adding a new trait of a similar type. On the other hand, those traits should now cancel each other instead of producing undefined behavior when chosen together, so it should be safe to remove them.

Reply:
But it goes against UX at the same time. User has to know what they this two or more traits are in conflict with each other. Without this, users can't understand it and will spend their points in something what cant' stack.